src/commands/lambda/prompt.ts

```diff
@@ -0,0 +1,195 @@
+import {blueBright, bold} from 'chalk'
+import {
+  CheckboxQuestion,
+  ConfirmQuestion,
+  InputQuestion,
+  ListQuestion,
+  prompt,
+  QuestionCollection,
+  Separator,
+} from 'inquirer'
+import {
+  AWS_ACCESS_KEY_ID_ENV_VAR,
+  AWS_ACCESS_KEY_ID_REG_EXP,
+  AWS_DEFAULT_REGION_ENV_VAR,
+  AWS_REGIONS,
+  AWS_SECRET_ACCESS_KEY_ENV_VAR,
+  AWS_SECRET_ACCESS_KEY_REG_EXP,
+  CI_API_KEY_ENV_VAR,
+  CI_API_KEY_SECRET_ARN_ENV_VAR,
+  CI_KMS_API_KEY_ENV_VAR,
+  CI_SITE_ENV_VAR,
+  DATADOG_API_KEY_REG_EXP,
+  SITES,
+} from './constants'
+import {sentenceMatchesRegEx} from './functions/commons'
+
+const awsCredentialsQuestions: QuestionCollection = [
+  {
+    // AWS_ACCESS_KEY_ID question
+    message: 'Enter AWS Access Key ID:',
+    name: AWS_ACCESS_KEY_ID_ENV_VAR,
+    type: 'input',
+    validate: (value) => {
+      if (!value || !sentenceMatchesRegEx(value, AWS_ACCESS_KEY_ID_REG_EXP)) {
+        return 'Enter a valid AWS Access Key ID.'
+      }
+
+      return true
+    },
+  },
+  {
+    // AWS_SCRET_ACCESS_KEY_ENV_VAR question
+    mask: true,
+    message: 'Enter AWS Secret Access Key:',
+    name: AWS_SECRET_ACCESS_KEY_ENV_VAR,
+    type: 'password',
+    validate: (value) => {
+      if (!value || !sentenceMatchesRegEx(value, AWS_SECRET_ACCESS_KEY_REG_EXP)) {
+        return 'Enter a valid AWS Secret Access Key.'
+      }
+
+      return true
+    },
+  },
+  {
+    // AWS_DEFAULT_REGION
+    choices: AWS_REGIONS,
+    default: 0,
+    message: 'Select an AWS region:',
+    name: AWS_DEFAULT_REGION_ENV_VAR,
+    type: 'list',
+  },
+]
+
+export const datadogApiKeyTypeQuestion = (datadogSite: string): ListQuestion => ({
+  choices: [
+    {
+      name: `Plain text ${bold('API Key')} (Recommended for trial users) `,
+      value: {
+        envVar: CI_API_KEY_ENV_VAR,
+        message: 'API Key:',
+      },
+    },
+    new Separator(),
+    {
+      name: `AWS Key Management Service ${bold('(KMS) API Key')}`,
```

Comment:
I would describe this as an "API key encrypted with AWS Key Management Service (KMS)".